<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0">
  <channel>
    <title>OpenAI 开发者论坛 - 最新帖子</title>
    <link>https://community.openai.com</link>
    <description>最新帖子</description>
    <lastBuildDate>Fri, 19 Apr 2024 09:20:48 GMT</lastBuildDate>
    <item>
      <title>Assistant API 总是返回空注释</title>
      <link>https://community.openai.com/t/assistant-api-always-return-empty-annotations/489285?page=3#post_47</link>
      <description><![CDATA[是的，这是一个很容易的误解，因为整个主题是“返回空注释”  抱歉。
您介意记录并发送扩展的 file_引用吗？因为我不确定该对象是否同时具有 file_ID 和 quote，或者只有 file_ID。
我真的很感激，所以我可以理解我的团队是否是唯一遇到这种合法注释中缺少引用问题的人（因为我在论坛上没有找到任何东西），或者这根本不是其他人正在使用的功能。]]></description>
      <guid>https://community.openai.com/t/assistant-api-always-return-empty-annotations/489285?page=3#post_47</guid>
      <pubDate>Fri, 19 Apr 2024 09:11:07 GMT</pubDate>
    </item>
    <item>
      <title>需要 OpenAI 使用给定的产品信息为品牌社交媒体生成内容</title>
      <link>https://community.openai.com/t/need-openai-to-generation-of-content-for-brands-social-media-using-given-product-information/708656#post_5</link>
      <description><![CDATA[我们提供数字产品设计和制作开发服务。我们的团队已经构建了几个生成式人工智能产品，例如 thenexed.com 。这篇文章的目的是尽量减少我们的研究。其他人建议或正在使用什么模型来创建此类应用程序。
谢谢@grandell1234]]></description>
      <guid>https://community.openai.com/t/need-openai-to-generation-of-content-for-brands-social-media-using-given-product-information/708656#post_5</guid>
      <pubDate>Fri, 19 Apr 2024 09:06:20 GMT</pubDate>
    </item>
    <item>
      <title>使用openai选择标题的代码</title>
      <link>https://community.openai.com/t/code-for-title-selection-using-openai/722138#post_1</link>
      <description><![CDATA[加载您的数据集
df = pd.read_excel(“/content/sample_data/report.xlsx”)
df = df.dropna(subset=[“desc”])
df = df.drop_duplicates(subset=[“desc”])
df = df.reset_index(drop=True)
清理文本数据
def cleantwt(twt):
if isinstance(twt, str):
twt = re.sub(“RT”, &quot; &quot;, twt)
twt = re.sub(”#[A-Za_z0-9]+“, “”, twt)
twt = re.sub(r&#39;[0-9]+&#39;, &#39;&#39;, twt)
twt = re.sub(&#39;[0123456789]+&#39;,&#39;&#39;,twt)
twt = re.sub(r&#39;[A-Za-z]&#39;, &#39;&#39;, twt)
twt = re.sub(“‘”“, “”, twt)
twt = re.sub(”\n”, “”, twt)
twt = re.sub(“https?://\S+”, “”, twt)
twt = re.sub(‘@[\S]‘, ‘’, twt)
twt = re.sub(&#39; [1]+|[\s]+&#39;, &#39;&#39;, twt)
        twt = re.sub(r&#39;[+&amp;+;+]|[&gt;+]|[Ù+]|[?+ ]|[“”]|@&#39;, &#39; &#39;,twt)
返回twt
其他：
返回twt
df[“clean_text”] = df[“desc”].apply(cleantwt)
加载停用词
以 open(&#39;/content/sample_data/stopwords.txt&#39;, &#39;r&#39;,encoding=&#39;utf-8&#39;) 作为文件：
stopwords_ur = file.read().splitlines()
删除停用词
def remove_stopwords(text):
返回 [如果单词不在 stopwords_ur 中，则返回文本中的单词]
标记并删除停用词
def 预处理(文本):
令牌 = gensim.utils.simple_preprocess(text)
tokens_no_stopwords = remove_stopwords(tokens)
返回 tokens_no_stopwords
应用预处理
df[“clean_text_no_stopwords”] = df[“clean_text”].apply（预处理）
创建 N-克
def make_n_grams(文本):
bigram = gensim.models.Phrases(文本, min_count=5, 阈值=100)
bigram_mod = gensim.models.phrases.Phraser(bigram)
trigram = gensim.models.Phrases(bigram[文本], 阈值=100)
trigram_mod = gensim.models.phrases.Phraser(trigram)
bigrams_text = [bigram_mod[doc] 用于文本中的文档]
trigrams_text = [trigram_mod[bigram_mod[doc]] 用于 bigrams_text 中的文档]
返回 trigrams_text
应用 N-克
data_ngrams = make_n_grams(df[“clean_text_no_stopwords”])
data_strings = [&quot; &quot;.join(doc) for data_ngrams 中的文档]
id2word = corpora.Dictionary(data_ngrams)
文本 = data_ngrams
corpus = [id2word.doc2bow(text) 用于文本中的文本]
在此之后的代码


\s ↩︎

]]></description>
      <guid>https://community.openai.com/t/code-for-title-selection-using-openai/722138#post_1</guid>
      <pubDate>Fri, 19 Apr 2024 08:59:44 GMT</pubDate>
    </item>
    <item>
      <title>修复“嗯...似乎出了问题”错误</title>
      <link>https://community.openai.com/t/fix-hmm-something-seems-to-have-gone-wrong-error/543179?page=5#post_100</link>
      <description><![CDATA[我已经解决这个问题两天了]]></description>
      <guid>https://community.openai.com/t/fix-hmm-something-seems-to-have-gone-wrong-error/543179?page=5#post_100</guid>
      <pubDate>Fri, 19 Apr 2024 08:54:10 GMT</pubDate>
    </item>
    <item>
      <title>在 ChatGPT 4 plus 中如何访问修复和修复</title>
      <link>https://community.openai.com/t/in-chatgpt-4-plus-how-do-i-access-inpainting-and-outpainting/496960#post_5</link>
      <description><![CDATA[很快是一个相当抽象的信息......
我本可以在我的 Fooocus 实例中完成此操作，但我支付了更好的渲染费用。现在，我觉得我浪费了我的钱。]]></description>
      <guid>https://community.openai.com/t/in-chatgpt-4-plus-how-do-i-access-inpainting-and-outpainting/496960#post_5</guid>
      <pubDate>Fri, 19 Apr 2024 08:53:54 GMT</pubDate>
    </item>
    <item>
      <title>修复“嗯...似乎出了问题”错误</title>
      <link>https://community.openai.com/t/fix-hmm-something-seems-to-have-gone-wrong-error/543179?page=5#post_99</link>
      <description><![CDATA[从昨天下午开始我就遇到同样的错误 - GPT-4 不起作用。 （团队帐号）]]></description>
      <guid>https://community.openai.com/t/fix-hmm-something-seems-to-have-gone-wrong-error/543179?page=5#post_99</guid>
      <pubDate>Fri, 19 Apr 2024 08:51:57 GMT</pubDate>
    </item>
    <item>
      <title>嗯...似乎出了问题 - 错误</title>
      <link>https://community.openai.com/t/hmm-something-seems-to-have-gone-wrong-error/721937#post_3</link>
      <description><![CDATA[我也有同样的问题，这是我发现的：

ChatGPT 3.5 仍然有效。
ChatGPT 4 给出错误“嗯……似乎出了问题。”。
ChatGPT 4 (ODD) 如果您上传图像并附上问题，它将正常响应。 （？！）
ChatGPT 4 Android 应用程序出现错误“服务器出现问题”。
]]></description>
      <guid>https://community.openai.com/t/hmm-something-seems-to-have-gone-wrong-error/721937#post_3</guid>
      <pubDate>Fri, 19 Apr 2024 08:51:45 GMT</pubDate>
    </item>
    <item>
      <title>如何在不发送提示的情况下向 gpt 提供/查看实时数据</title>
      <link>https://community.openai.com/t/how-to-feed-view-real-time-data-to-gpt-without-sending-a-prompt/722038#post_3</link>
      <description><![CDATA[使用函数调用。仅当需要时才会调用获取实时行情数据或新闻文章的函数。]]></description>
      <guid>https://community.openai.com/t/how-to-feed-view-real-time-data-to-gpt-without-sending-a-prompt/722038#post_3</guid>
      <pubDate>Fri, 19 Apr 2024 08:49:29 GMT</pubDate>
    </item>
    <item>
      <title>修复“嗯...似乎出了问题”错误</title>
      <link>https://community.openai.com/t/fix-hmm-something-seems-to-have-gone-wrong-error/543179?page=5#post_98</link>
      <description><![CDATA[连续两天。 ChatGPT 3.5 运行良好。每当我尝试使用 ChatGPT 4 时，它都会说：“嗯……似乎出了问题。”。
我正在考虑取消订阅，因为我无法使用我所支付的费用。]]></description>
      <guid>https://community.openai.com/t/fix-hmm-something-seems-to-have-gone-wrong-error/543179?page=5#post_98</guid>
      <pubDate>Fri, 19 Apr 2024 08:39:28 GMT</pubDate>
    </item>
    <item>
      <title>AttributeError：“Beta”对象没有属性“vector_stores”</title>
      <link>https://community.openai.com/t/attributeerror-beta-object-has-no-attribute-vector-stores/721461#post_6</link>
      <description><![CDATA[谢谢本。经过进一步调查，问题似乎与 Streamlit 有一些冲突（我正在构建 Streamlit 应用程序）。当我创建一个没有streamlit的测试项目时，vector_store创建得很好。
莱恩]]></description>
      <guid>https://community.openai.com/t/attributeerror-beta-object-has-no-attribute-vector-stores/721461#post_6</guid>
      <pubDate>Fri, 19 Apr 2024 08:38:20 GMT</pubDate>
    </item>
    <item>
      <title>助手创建线程，上传文件出错</title>
      <link>https://community.openai.com/t/assistants-create-threads-and-an-error-occurs-when-uploading-files/722087#post_3</link>
      <description><![CDATA[


 sps：
&lt;块引用&gt;
[{&quot;type&quot;: &quot;file_search&quot;}] 


非常感谢您的回复。
不过，我已经升级到最新版本了。openai@4.38.1。 
以下是有关线程的文档： Doc 
如果我这样写
&quot;附件&quot;: [
        { &quot;file_id&quot;: message_file.id, &quot;tools&quot;: [{&quot;type&quot;: &quot;file_search&quot;}] }
      ],

，仍然会出现类型错误。
TS2322: 类型 &#39;{ file_id: any;工具：{ 类型：字符串； }[]; }[] |未定义&#39; 不可分配给类型&#39;Attachment[] |空 |不明确的&#39;。输入 &#39;{ file_id: 任意;工具：{ 类型：字符串； }[]; }[]&#39; 不可分配给类型“Attachment[]”。输入 &#39;{ file_id: 任意;工具：{ 类型：字符串； }[]; }&#39; 不可分配给类型“附件”。属性“工具”的类型不兼容。类型 &#39;{ 类型：字符串； }[]&#39; 不可分配给类型 &#39;(&quot;file_search&quot; | &quot;code_interpreter&quot;)[]&#39;。类型 &#39;{ 类型：字符串； }&#39; 不可分配给类型 &#39;&quot;file_search&quot; | “代码解释器”&#39;。
]]></description>
      <guid>https://community.openai.com/t/assistants-create-threads-and-an-error-occurs-when-uploading-files/722087#post_3</guid>
      <pubDate>Fri, 19 Apr 2024 08:37:48 GMT</pubDate>
    </item>
    </channel>
</rss>